### DISCUSSION TITLE
Textue corruptjon on macos

### DISCUSSION THREAD

**Discussion by PinkPlague** [ppy/osu#19267] [Q&A] [answered]
### Type

Cosmetic

### Bug description

I tried launching version 2021.1225.0 on MacOS X 10.13.6 and all text excluding title text, user name text, and some buttons text, were just odd looking lines and squares. on top of all folders inside the app being reverted to random letters and characters.

### Screenshots or videos

![Screen Shot 2022-07-20 at 11 21 40 PM](https://user-images.githubusercontent.com/104692088/180135746-4b82deee-dfce-4825-a7a9-8447eb2f1ac8.png)
![Screen Shot 2022-07-20 at 11 21 34 PM](https://user-images.githubusercontent.com/104692088/180135755-3608fbe6-b20f-4ab9-ba91-a0ecac2de677.png)
![Screen Shot 2022-07-20 at 11 21 27 PM](https://user-images.githubusercontent.com/104692088/180135759-9939f0a4-7369-4bc0-b0ea-c45cbd9d265c.png)
![Screen Shot 2022-07-20 at 11 21 19 PM](https://user-images.githubusercontent.com/104692088/180135760-b8815f62-7d3a-4ac3-a3c4-d90c10fa2373.png)
![Screen Shot 2022-07-20 at 11 21 10 PM](https://user-images.githubusercontent.com/104692088/180135766-0750f173-c254-4cc4-9d17-226f56c813c5.png)

### Version

2021.1225.0

### Logs

(NOTE: I could not navigate to the logs option due to most text being bugged)

**Comment by frenzibyte** [comment]
Here's where the "Open osu! folder" button resides:

<img width="1708" alt="CleanShot 2022-07-21 at 08 26 05@2x" src="https://user-images.githubusercontent.com/22781491/180136184-a6c1bc74-2d78-40cf-b07e-0320061e370d.png">

**Reply by PinkPlague** [reply]
The options menu would not load, so that rules out this

**Reply by frenzibyte** [reply]
Check out `~/Library/Applcation Support/osu` or `~/.local/share/osu`, depending on when you've started playing osu!.

**Reply by PinkPlague** [reply]
Managed to get it to load, here are the logs
[database.log](https://github.com/ppy/osu/files/9174815/database.log)
[performance.log](https://github.com/ppy/osu/files/9174817/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9174818/runtime.log)
.

**Reply by PinkPlague** [reply]
I'm kind of worried that the "network logs" file has my ip

**Reply by frenzibyte** [reply]
All logs will never have any of your private informations printed, the `network.log` only contains logs about status of network requests to know if something has failed or otherwise, but it never contains the details of the request.

**Reply by PinkPlague** [reply]
ok, by the way the version I am running is the latest version I can run without not being able to run it.

**Comment by frenzibyte** [comment, marked as answer]
In addition, this should be a duplicate of https://github.com/ppy/osu/issues/9314, but would be helpful to know what GPU you're using via game logs.

**Reply by PinkPlague** [reply]
this doesn't not inform me on how to fix this bug, I am new to the community and just want to play OSU! while away from my pc.

**Reply by frenzibyte** [reply]
Unfortunately you've ended up on an issue that we've been tracking for a long time now, and no solution was found as far as I've seen.
